src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  width: 200px;
+  height: 408px;
+}
+
+.product {
+  padding: 32px 16px 16px;
+
+  &__photo {
+    position: center;
+    width: 100%;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $main-accent;
+  margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__rank {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: auto;
+
+    &-stars {
+      display: flex;
+
+      &-star {
+        width: 16px;
+        height: 16px;
+        background-image: url(../images/star.svg);
+        background-repeat: no-repeat;
+        background-position: center;
+      }
+    
+      & :nth-child(-n+4) {
+        background-image: url(../images/star-active.svg);
+      }
+
+    }
+
+    &-reviews {
+      color: $main-accent;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary;
+    display: flex;
+    justify-content: space-between;
+    gap: auto;
+    align-items: center;
+
+      &-value {
+        color: $main-accent;
+        text-align: right;
+        font-size: 16px;
+        font-weight: 700;
+
+      }
+  }
+}
+
+.btn {
+  display: inline-block;
+  text-align: center;
+  width: 100%;
+  height: 40px;
```

Comment:
it'a not a good practice to hardcode height for elements with content inside. use padding instead